### PR TITLE
Update docs links

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Last their is an Interface for every type of class or abstract implementation th
 
 ## Documentation
 
-There is generated API documentation for all tags and released versions. Those can be found at [flint.github.com/Stampie/api/master](http://flint.github.com/Stampie/api/master).
+There is generated API documentation for all tags and released versions. Those can be found at [stampie.github.io/Stampie/api/master/](http://stampie.github.io/Stampie/api/master/).
```
Comment (stof) [inline README.md]: I would use HTTPS for the link

Comment (Nyholm) [inline README.md]: Thank you

File: README.md [hunk 2]
```diff
@@ -57,11 +57,11 @@ Stampie is itself completly decoupled and does not depend on any framework.
 
 ### Integrations
 
-* [HBStampieBundle](http://github.com/henrikbjorn/HBStampieBundle) it is also [on packagist](http://packagist.org/packages/henrikbjorn/stampie-bundle)
+* [HBStampieBundle](https://github.com/henrikbjorn/HBStampieBundle) it is also [on packagist](https://packagist.org/packages/henrikbjorn/stampie-bundle)
 
 ## Testing
 
-Stampie is [Continuous Integration](http://en.wikipedia.org/wiki/Continuous_integration) tested with [Travis](http://travis-ci.org) and aims for a high coverage percentage.
+Stampie is [Continuous Integration](httsp://en.wikipedia.org/wiki/Continuous_integration) tested with [Travis](https://travis-ci.org) and aims for a high coverage percentage.
```
Comment (Ninir) [inline README.md]: Small typo there ;)

Comment (Nyholm) [inline README.md]: thanks

File: README.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
 # Stampie
 
-[![Build Status](https://secure.travis-ci.org/henrikbjorn/Stampie.png)](http://travis-ci.org/henrikbjorn/Stampie)
+[![Build Status](https://secure.travis-ci.org/henrikbjorn/Stampie.png)](https://travis-ci.org/Stampie/Stampie)
```
Comment (stof) [inline README.md]: I would switch to the SVG version of the badge (and stop using the old `secure` subdomain)

File: README.md [hunk 4]
```diff
@@ -57,11 +57,11 @@ Stampie is itself completly decoupled and does not depend on any framework.
 
 ### Integrations
 
-* [HBStampieBundle](http://github.com/henrikbjorn/HBStampieBundle) it is also [on packagist](http://packagist.org/packages/henrikbjorn/stampie-bundle)
+* [HBStampieBundle](https://github.com/henrikbjorn/HBStampieBundle) it is also [on packagist](https://packagist.org/packages/henrikbjorn/stampie-bundle)
```
Comment (stof) [inline README.md]: URL should be updated, as the bundle repo is not in the Stampie org too

**Comment by Nyholm** [comment]
Thank you for the reviews. I have updated the PR accordingly.

**Pull request merged**